lib/navigation.js

```diff
@@ -73,6 +80,42 @@ module.exports = (config) => (files, metalsmith, done) => {
   done()
 }
 
+class NavigationItem {
+  constructor(data) {
+    Object.assign(this, data)
```

Comment:
*note* Rename to `page` and delegate access to `this.data` rather than assigning all its properties to the `NavigationItem` so the NavigationItem's properties are clearly defined.